openapi-generator/src/main/kotlin/org/jellyfin/openapi/builder/openapi/OpenApiApiServicesBuilder.kt

```diff
@@ -63,6 +64,39 @@ class OpenApiApiServicesBuilder(
 		else -> null
 	}
 
+	private fun buildRequestBody(requestBody: RequestBody, serviceName: String, operationName: String): TypeName? {
+		// Filter out duplicate JSON values
+		val requestBodyTypes = requestBody.content.keys.filterNot {
+			MimeType.IGNORED_JSON_TYPES.contains(it)
+		}
+
+		// Check amount of types
+		if (requestBodyTypes.size > 1) throw OpenApiGeneratorError("Multiple request body types are unsupported.")
+		else if (requestBodyTypes.isEmpty()) return null
+
+		// Retrieve info
+		val requestBodyType = requestBodyTypes.first()
```

Comment:
Could maybe merge these into a neat `when` on `requestBodyTypes.size` for readability, like
```kt
		// Assert count and retrieve type
		val requestBodyType = when (requestBodyTypes.size) {
			0 -> return null
			1 -> requestBodyTypes.first()
			else -> throw OpenApiGeneratorError("Multiple request body types are unsupported.")
		}
```